packages/strapi-admin/domain/action.js

```diff
@@ -10,24 +10,29 @@ const actionFields = [
   'pluginName',
   'subjects',
   'conditions',
+  'options',
 ];
 
+const defaultAction = {
+  options: {
+    fieldsGranularity: true,
```

Comment:
Not sure about the naming for this one.
Do you have a preference @alexandrebodin?